app/views/foreman_tasks/recurring_logics/index.html.erb

```diff
@@ -5,6 +5,11 @@
 <%= alert(:class => 'alert-info', :id => 'multiple-alert', :close => false, :header => '', :text => @errors) %>
 <% end %>
 
+<% title_actions link_to(_('Clear Cancelled'),
+            clear_cancelled_foreman_tasks_recurring_logics_path,
+            class:  ['btn', 'btn-sm', 'btn-primary'].compact,
+            method: :post ) %>
```

Comment:
The button should not be shown if the user doesn't have the `edit_recurring_logics` permission assigned. For reference take a look at https://github.com/theforeman/foreman_remote_execution/blob/master/app/helpers/remote_execution_helper.rb#L69

Reply:
Also that `.compact` doesn't really do anything there so you can drop it

Reply:
Thanks for the explanation, fixed